test/int/integration_suite_test.go

```diff
@@ -107,8 +111,13 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	Expect(err).ToNot(HaveOccurred())
 	Expect(cfg).ToNot(BeNil())
 
+	var b bytes.Buffer
+	e := gob.NewEncoder(&b)
+	err = e.Encode(*cfg)
```

Comment:
this config has all the authentication pre-configured, we can pass this to the other nodes so we no longer encounter the x509 errors.